### PR TITLE
Refactor and additions

### PR DESCRIPTION
**Pull request by brandonsturgeon**
Tested, looks good!

### CODE REVIEW

File: lua/autorun/client/cl_cfc_help.lua [hunk 1]
```diff
@@ -1,6 +1,47 @@
-local uiColor = Color( 36, 41, 67, 255 )
+CFCHelp = {
+    colors = {
+        ui = Color( 36, 41, 67, 255 ),
+        lightBlue = Color( 83, 227, 251, 255 ),
+        lightGreen = Color( 83, 251, 191 ),
+        white = Color( 255, 255, 255, 255 ),
+        grey = Color( 175, 175, 175, 255 )
```
Comment (samuelWilliams99) [inline lua/autorun/client/cl_cfc_help.lua:7]: alpha of 255 is default, but if it must stay, probably add it to `lightGreen`

File: lua/autorun/client/cl_cfc_help.lua [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+    },
 
-local function openHelp()
+    commandMap = {
+        ["!help"] = {
+            helpType = "html",
+            url = "https://cfcservers.org/learn/player",
+            title = "CFC Help"
+        },
```
Comment (samuelWilliams99) [inline lua/autorun/client/cl_cfc_help.lua:15]: Seems help used to be a silent command, but now isnt? Is this an intentional change?

File: lua/autorun/client/cl_cfc_help.lua [hunk 3]
```diff
@@ -10,31 +51,41 @@ local function openHelp()
...
 
     if ply == LocalPlayer() then
-        openHelp()
+        local helpType = commandInfo.helpType
+
+        if helpType == "html" then
+            CFCHelp:openUrl( commandInfo.url, commandInfo.title )
+        end
+
+        if helpType == "func" then
```
Comment (samuelWilliams99) [inline lua/autorun/client/cl_cfc_help.lua:83]:
```suggestion
        if helpType == "function" then
```
Your data structure has this set as `function`

**Review by samuelWilliams99 — CHANGES_REQUESTED** [review]

File: lua/autorun/client/cl_cfc_help.lua (hunk 2 above)
Comment (brandonsturgeon) [reply lua/autorun/client/cl_cfc_help.lua:15]: Yeah I think it's better if people use the command, other people will see it exists and maybe take a peek

**Review by samuelWilliams99 — APPROVED** [review]

**Pull request merged**